docs/classad-attributes/machine-classad-attributes.rst

```diff
@@ -464,7 +464,17 @@ Machine ClassAd Attributes
     *condor_startd* expressions (and you would not want to).
 
 :classad-attribute-def:`LoadAvg`
-    A floating point number representing the current load average.
+    A floating point number representing the current load average over time.
+    This number goes up by 1.0 for every runnable thread.  More concretely, if
+    a single-core machine has a load average of 1.0, it means the one cpu is
+    fully utilized -- on average, there is one running thread at all times.  If
```

Comment:
Slacked this to you but will put it here for my review as well: 
This might make more sense with an `in other words,` , `which means,` , or a `more specifically,` if that is what is intended by the `--`